shared/readSpikeTrials_continuous.m

```diff
@@ -0,0 +1,134 @@
+function [SpikeTrials] = readSpikeTrials_continuous(cfg,MuseStruct,SpikeRaw,force)
```

Comment:
This function conflicts with intended workflow by starting with a musestruct with separate trials (that first has to be created) instead of just segmenting continuous spike data (or LFP data for that matter) into trials.

The problem I have with this way of doing it, is that MuseStruct is a structure made to interface with Muse markers. Not a generic trial definition, which is easily done in FieldTrip or like https://github.com/stephenwhitmarsh/EpiCode/blob/master/shared/readSpykingCircusPSG.m - line 185. I have mentioned this earlier, i.e. in relationship with Antoine's data.

To maintain transparency of the codebase I do not want MuseStruct to become anything but an interface with Muse. Didactically, and in terms of long-term integration with FieldTrip, it is important to maintain this difference as well.

Please move this function to your project directory. We can discuss how to do Welch method (cutting in shorter segments) in Louis' project, if you had that in mind, and instead start from e.g. readSpykingCircusPSG.